test/extended/cmd.sh

```diff
@@ -98,51 +99,43 @@ oc login ${MASTER_ADDR} -u pull-secrets-user -p password --certificate-authority
 
 # create a new project and push a busybox image in there
 oc new-project image-ns
-oc delete all --all
-IMAGE_NS_TOKEN=`oc get sa/builder --template='{{range .secrets}}{{ .name }} {{end}}' | xargs -n 1 oc get secret --template='{{ if .data.token }}{{ .data.token }}{{end}}' | base64 -d -`
-docker login -u imagensbuilder -p ${IMAGE_NS_TOKEN} -e fake@example.org ${DOCKER_REGISTRY}
-oc tag --source=docker busybox:latest image-ns/busybox:latest
-oc import-image busybox
-docker pull busybox
-docker tag -f docker.io/busybox:latest ${DOCKER_REGISTRY}/image-ns/busybox:latest
-docker push ${DOCKER_REGISTRY}/image-ns/busybox:latest
-docker rmi -f ${DOCKER_REGISTRY}/image-ns/busybox:latest
+os::cmd::expect_success "oc delete all --all"
+IMAGE_NS_TOKEN=`oc sa get-token builder`
+os::cmd::expect_success "docker login -u imagensbuilder -p ${IMAGE_NS_TOKEN} -e fake@example.org ${DOCKER_REGISTRY}"
```

Comment:
Ok, will revert in my pull. Naming of tests is much better without tokens in there.